netbox/resource_netbox_ipam_aggregate.go

```diff
@@ -63,7 +63,7 @@ func resourceNetboxIpamAggregate() *schema.Resource {
 			"description": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				Default:      " ",
+				Default:      "",
```

Comment:
We need to keep this in place to be sure that description is removed.

Reply:
Indeed, my mistake there. I'll remove these changes.